android/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:latest
+
+RUN apt-get update && \
+  apt-get install -y \
```

Comment:
Do we know if these are actually needed/can we whittle it down to the minimun? Some of these seem like they wouldn't be used and we can save CI time by not including them.

Reply:
Good point, i'll check

Reply:
9f5c7d8